src/ParallelAlgorithms/Events/ObserveTimeStep.hpp

```diff
@@ -71,9 +74,15 @@ struct FormatTimeOutput
                          const double /* slab_size */,
                          const double /* min_time_step */,
                          const double /* max_time_step */,
-                         const double /* effective_time_step */) const
+                         const double /* effective_time_step */,
+                         const double min_wall_time,
+                         const double max_wall_time) const
       noexcept {
-    return "Current time: " + std::to_string(time);
+    std::stringstream ss;
+    ss  << "Current time: " << std::to_string(time)
+        << "\nMin wall time: " << std::to_string(min_wall_time)
```

Comment:
What do you think about indenting the `Min` and `Max` lines a couple spaces so that it's easier to tell what they're associated with? Or maybe even keeping them on the same line as `Current time`? I'm not sure how long that line gets if all 3 are printed together. Maybe renaming `Current time` to `Sim time` would keep things short? Just some thoughts :)

Reply:
I also think a bit of indentation would be nice, how about:

```
Simulation time: 1.234
  Wall time: 4.678s (min) - 9.012s (max)
```

(or some variation of this)